_includes/footer.html

```diff
@@ -2,7 +2,84 @@
 <script src="https://code.jquery.com/jquery-3.5.1.min.js"
   integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
 <script type="text/javascript" src="{{ "/assets/bootstrap/js/bootstrap.min.js" | relative_url }}"></script>
+<script src="https://cdn.jsdelivr.net/npm/chart.js@3.7.1/dist/chart.min.js"></script>
 <script type="text/javascript">
+  function createChart(labels, dat) {
+    const ctx = $("#weeklyCommits");
+    const data = {
+      labels: labels,
+      datasets: [{
+        data: dat,
+        backgroundColor: "#FFAD33",
+        hoverBackgroundColor: "#FFAD33"
+      }]
+    };
+
+    const options = {
+      scales: {
+        y: {
+          ticks: {
+            beginAtZero: true,
+            color: "#FFAD33"
+          },
+          grid: {
+            drawBorder: false,
+            display: false
+          }
+        },
+        x: {
+          ticks: {
+            maxTicksLimit: (context) => {
+              return context.chart.width > 500 ? 18 : 9;
+            },
+            color: "#FFAD33"
+          },
+          grid: {
+            drawBorder: false,
+            display: false
+          }
+        }
+      },
+      plugins: {
+        title: {
+          display: true,
+          position: "top",
+          text: "Weekly Contributions",
+          color: "#FFAD33",
+          font: {
+            size: 18,
+          }
+        },
+        legend: {
+          display: false
+        },
+        tooltip: {
+          backgroundColor: "#FFAD33",
+          borderWidth: 1,
+          borderColor: "#1a2743",
+          titleColor: "#37528C",
+          titleMarginBottom: 0,
+          callbacks: {
+            title: (t) => {
+              return t[0].formattedValue + " commits the week of " + new Date(t[0].label).toLocaleString(undefined, {

```

Comment:
Same as https://github.com/ruffle-rs/ruffle-rs.github.io/pull/90/files#r897128526.

Reply:
I opted for an arrow function here as well. This one I was more unsure about, as it's a long statement, but it is still just a single line (and I'd argue it should be), and arrow-functions are most appropriate when using just a single line.